spaces/S000087/properties/P000062.md

```diff
@@ -0,0 +1,11 @@
+---
+space: S000087
+property: P000062
+value: true
+---
+
+Observe that a (countable) subcollection $\mathscr A$ of an open cover $\mathscr U$
+covering the subset $\{\omega_1\}\times[0,\omega)$ contains all but countably many columns of $[0,\omega_1]\times[0,\omega)$.
+Therefore its closure has countable complement $B$ in $X$.
+$\mathscr A$ can be then extended so that it is still countable and covers $B$.
+The union of the extended subcollection is dense.
```

Comment:
```suggestion
Given an open cover $\mathscr U$ of $X$, choose a countable subcollection $\mathscr A$ covering the countable set $\{\omega_1\}\times[0,\omega)$.
The union $\bigcup\mathscr A$ contains all but countably many columns of $[0,\omega_1]\times[0,\omega)$.
Therefore the closure of $\bigcup\mathscr A$ has countable complement $B$ in $X$.
One can then extend $\mathscr A$ to a countable subcollection of $\mathscr U$ that covers $B$.
The union of the extended subcollection is dense in $X$.
```
No problem with your idea, but this tweak seems a little easier to follow for me.
(cut and paste in the pi-base preview window to compare and see what you think)